gloo/common/CMakeLists.txt

```diff
@@ -4,6 +4,7 @@ set(GLOO_COMMON_SRCS
   )
 
 set(GLOO_COMMON_HDRS
+  "${CMAKE_CURRENT_SOURCE_DIR}/aligned_allocator.h"
```

Comment:
If this is linux specific, should we also move this into an if-else clause like what #199 did for `linux.h`?

Reply:
It's not Linux specific luckily, but `memalign` is.